src/Redux/books/booksSlice.js

```diff
@@ -0,0 +1,21 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  booklist: [],
+};
+
+const bookSlice = createSlice({
+  name: 'bookList',
+  initialState,
+  reducers: {
+    addBook: (state, action) => {
+      state.booklist.pop(action.payload);
+    },
+    removeBook: (state, action) => {
+      state.booklist.pop(action.payload);
+    },
```

Comment:
Great job @hasnatali1947 🤝

- [ ] Kindly use the filter method which books by their ID in order to remove a book from the collection.